<launch>
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_local" clear_params="true">
        <rosparam command="load" file="$(find kamayuc_localization)/params/ekf_local.yaml" />
        <remap from="odometry/filtered" to="ekf_local/odometry"/>
    </node>
</launch>
